doc/api/buffer.markdown

```diff
@@ -1042,7 +1042,7 @@ console.log(`${len} bytes: ${buf.toString('utf8', 0, len)}`);
 * `value` {Number} Bytes to be written to Buffer
 * `offset` {Number} `0 <= offset <= buf.length - 8`
 * `noAssert` {Boolean} Default: false
-* Return: {Number} Numbers of bytes written
+* Return: {Number} The offset plus the number of written bytes
```

Comment:
Perhaps 'offset' should be surrounded by backticks in each of these changes since it's referencing a parameter.